zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -898,6 +898,10 @@ public String toJson() {
   }
 
   public static Note fromJson(String json) {
+    GsonBuilder gsonBuilder = new GsonBuilder();
+    gsonBuilder.setPrettyPrinting();
+    Gson gson = gsonBuilder.registerTypeAdapter(Date.class, new NotebookImportDeserializer())
```

Comment:
I have made a few changes, could you please review and let me know if anything else needs to be changed? Making any other changes seems to give me checkstyle errors.